python/setup.py

```diff
@@ -53,7 +53,7 @@ def find_packages_by_root_package(where):
     version = "4.1.0",
     author = "Jens Engel",
     author_email = "jenisys@noreply.github.com",
-    url = "https://github.com/cucumber/tag-expressions-python",
+    url = "https://github.com/cucumber/tag-expressions",
```

Comment:
Will that work with the python package being in the `python` subfolder?

Reply:
Oh, this is "just" the link to the homepage!? Nothing related to installing the package like it may have been with go!?